### DISCUSSION TITLE
Thank You For The Continued Support

### DISCUSSION THREAD

**Discussion by SoundsSerious** [matplotlib/basemap#586] [General]
I'm a long time user of basemap, and have come to rely on this library. I want to thank all the folks who are continuing to maintain it and support new features.

**Comment by molinav** [comment]
Thank you for your feedback, @SoundsSerious! The development might be a bit slow at times, but we try to keep the library as stable as possible and in shape. 👍

**Reply by SoundsSerious** [reply]
As I learn a bit more about the internals hopefully I can make a contribution, to be honest however i dont fully understand the basemap stack or why it the author wanted to depreciate it.
